RocketChat/Rocket.Chat
# fix: Rooms, teams and roles LDAP syncs are not triggered on login

## Proposed changes (including videos or screenshots)

 - Fixed rooms, teams and roles LDAP syncs not running on login (even when the **"Update User Data on Login"** setting is enabled) due to RC being binded (authenticated) to the logged in user instead of the LDAP admin.

## Issue(s)


## Steps to test or reproduce

LDAP rooms, teams and roles syncs currently only work on background sync or when hitting the "Sync now" button. All of them should now be triggered on login in case the **"Update User Data on Login"** setting is enabled (**LDAP > Data Sync**)

## Further comments

[CORE-437](https://rocketchat.atlassian.net/browse/CORE-437)


[CORE-437]: https://rocketchat.atlassian.net/browse/CORE-437?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
